saltgui/static/scripts/api.js

```diff
@@ -113,6 +113,12 @@ class API {
       eauth: "pam"
     };
 
+    // overrule the eauth method when one is selected
+    var type = document.querySelector("#login-form #eauth");
+    if(type.value !== "default")
+      params.eauth = type.value;
```

Comment:
Could you use brackets or change this as one line?

Reply:
done